great_expectations/checkpoint/actions.py

```diff
@@ -670,6 +691,13 @@ def run(
             use_ssl=self.use_ssl,
         )
 
+        checkpoint = checkpoint_result.checkpoint_config
```

Comment:
Non-blocking, but any reason not to do analytics for all actions, and then have this event submit be done by whatever invokes actions? Seems simpler and more robust.

Reply:
The issue is that `notify_on` is only present on certain objects - `UpdateDataDocs` causes some weirdness

Reply:
Gotcha. Figured it was something like that!